kubernetes/samples/scripts/create-oracle-db-service/README.md

```diff
@@ -0,0 +1,63 @@
+# Creating an Oracle database service and RCU schema for a Fusion Middleware domain
+
+The sample scripts in this directory demonstrate how to:
+* Start an Oracle database (DB) service in a Kubernetes cluster.
+* Stop an Oracle DB service in a Kubernetes cluster.
+
+## Start an Oracle database service in a Kubernetes cluster
```

Comment:
Same

Reply:
Modified all the instance of database to Database